auto_round/model_info.py

```diff
@@ -0,0 +1 @@
+SPECIAL_ATTENTION_LIST = ["Baichuan2-13B-Chat"]
```

Comment:
if chat has issue, I guess Baichuan2-13B also has the issue. Baichuan should have some other model size, so we should also have a check
Another way is using try catch, if normal forward has some issue, we try special attention way